<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorOpacityEditorWidget</class>
 <widget class="QWidget" name="ColorOpacityEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="pqTransferFunctionWidget" name="OpacityEditor" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="pqTransferFunctionWidget" name="GradientOpacityEditor" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="pqTransferFunctionWidget" name="ColorEditor" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QvisGaussianOpacityBar" name="GaussianOpacityEditor" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Qvis2DTransferFunctionWidget" name="TwoDTransferFunction" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QToolButton" name="ResetRangeToData">
         <property name="toolTip">
          <string>Rescale to data range</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../Components/Resources/pqComponents.qrc">
           <normaloff>:/pqWidgets/Icons/pqResetRange24.png</normaloff>:/pqWidgets/Icons/pqResetRange24.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ResetRangeToCustom">
         <property name="toolTip">
          <string>Rescale to custom range</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../Components/Resources/pqComponents.qrc">
           <normaloff>:/pqWidgets/Icons/pqResetRangeCustom24.png</normaloff>:/pqWidgets/Icons/pqResetRangeCustom24.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ResetRangeToDataOverTime">
         <property name="toolTip">
          <string>Rescale to data range over all timesteps</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../Components/Resources/pqComponents.qrc">
           <normaloff>:/pqWidgets/Icons/pqResetRangeTemporal24.png</normaloff>:/pqWidgets/Icons/pqResetRangeTemporal24.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="InvertTransferFunctions">
         <property name="toolTip">
          <string>Invert the transfer functions</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../Components/Resources/pqComponents.qrc">
           <normaloff>:/pqWidgets/Icons/pqInvert24.png</normaloff>:/pqWidgets/Icons/pqInvert24.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="DisableOpacityGradient">
         <property name="toolTip">
          <string>Disable Gradient Opacity</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../Components/Resources/pqComponents.qrc">
           <normaloff>:/pqWidgets/Icons/PluginRed.png</normaloff>
           <normalon>:/pqWidgets/Icons/PluginGreen.png</normalon>:/pqWidgets/Icons/PluginRed.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="ChoosePreset">
         <property name="toolTip">
          <string>Choose preset</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../Components/Resources/pqComponents.qrc">
           <normaloff>:/pqWidgets/Icons/pqFavorites16.png</normaloff>:/pqWidgets/Icons/pqFavorites16.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="SaveAsPreset">
         <property name="toolTip">
          <string>Save to preset</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../Components/Resources/pqComponents.qrc">
           <normaloff>:/pqWidgets/Icons/pqSave16.png</normaloff>:/pqWidgets/Icons/pqSave16.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="Edit">
         <property name="toolTip">
          <string>Manually edit transfer functions</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../Components/Resources/pqComponents.qrc">
           <normaloff>:/pqWidgets/Icons/pqAdvanced26.png</normaloff>:/pqWidgets/Icons/pqAdvanced26.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Data:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="pqLineEdit" name="CurrentDataValue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="UseLogScale">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When checked, the mapping from data to colors is done using a log-scale. Note that this does not affect the mapping of data to opacity.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Use log scale when mapping data to colors</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="EnableOpacityMapping">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When checked, the opacity function is used to render translucent surfaces.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Enable opacity mapping for surfaces</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="AutoRescaleRange">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When checked, the transfer function will be rescaled using updated data ranges on Apply&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Automatically rescale transfer functions to fit data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqTransferFunctionWidget</class>
   <extends>QWidget</extends>
   <header>pqTransferFunctionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pqLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QvisGaussianOpacityBar</class>
   <extends>QWidget</extends>
   <header>Qvis/QvisGaussianOpacityBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Qvis2DTransferFunctionWidget</class>
   <extends>QWidget</extends>
   <header>Qvis/Qvis2DTransferFunctionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
